the-internet/test-output/junitreports/TEST-restassured.Homework_4_PATCH.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated by org.testng.reporters.JUnitReportReporter -->
+<testsuite hostname="Josephs-MacBook-Pro-3.local" failures="0" tests="1" name="restassured.Homework_4_PATCH" time="1.592" errors="0" timestamp="2020-11-16T18:12:55 EST" skipped="0">
+  <testcase classname="restassured.Homework_4_PATCH" name="test_1_put" time="1.592"/>
+  <system-out/>
```

Comment:
It's difficult to assert timestamp because the timestamp is when that record was written to the DB, not when you submitted the POST. Best practice is to get the current timestamp then ensure it was changed after you POSTed.